source/interface/transport_interface.h

```diff
@@ -195,14 +195,11 @@ typedef struct NetworkContext NetworkContext_t;
  * @transportcallback
  * @brief Transport interface for receiving data on the network.
  *
- * @note It is RECOMMENDED that the transport receive implementation
- * does NOT block when requested to read a single byte. A single byte
- * read request can be made by the caller to check whether there is a
- * new frame available on the network for reading.
- * However, the receive implementation MAY block for a timeout period when
- * it is requested to read more than 1 byte. This is because once the caller
- * is aware that a new frame is available to read on the network, then
- * the likelihood of reading more than one byte over the network becomes high.
+ * @note It is HIGHLY RECOMMENDED that the transport receive
+ * implementation does NOT block.
+ * coreMQTT will continue to call the transport interface if it receives
+ * a partial packet until it accumulates enough data to get the complete
+ * MQTT packet.
```

Comment:
Is this supposed to reference coreMQTT?

Reply:
NOTE: To "fix" this would require a change in coreMQTT as this [commit](https://github.com/FreeRTOS/coreMQTT/commit/7b68936daf1e177b6b19682eb8bcd1d13d3f774c) changed the file to directly reference the library. coreMQTT is the source of truth for it.

This will be done in a later PR as it is outside the scope of this PR

Reply:
The SSOT check forces it to be same. We can reword this later in both repositories as we discussed :)